Fix for Apple platforms:
* Ensure install_name with @rpath token
* Keep rpath clean in installed binaries

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 # CMake build script for ZeroMQ
 project(ZeroMQ)
 
-if(${CMAKE_SYSTEM_NAME} STREQUAL Darwin)
+if(1)
   cmake_minimum_required(VERSION 3.0.2)
 else()
   cmake_minimum_required(VERSION 2.8.12)
@@ -44,7 +44,7 @@ if (ENABLE_INTRINSICS)
    add_definitions(-DZMQ_HAVE_ATOMIC_INTRINSICS)
 endif()
 
-if(${CMAKE_SYSTEM_NAME} STREQUAL Darwin)
+if(0)
   # Find more information: https://cmake.org/Wiki/CMake_RPATH_handling
 
   # Apply CMP0042: MACOSX_RPATH is enabled by default
